corehq/apps/export/views/list.py

```diff
@@ -233,7 +233,7 @@ def _get_odata_url(self, export):
                 'domain': export.domain,
                 'api_name': 'v0.5',
                 'resource_name': resource_class._meta.resource_name,
-                'pk': export.get_id,
+                'pk': export.get_id + '/feed',
```

Comment:
what's this update about?

Reply:
For some reason this is necessary - or else it won't include the ```/feed``` part at the end of ```/config_id/feed```.  ```/config_id/``` points to the service doc, and ```config_id/feed``` to the data feed.

Reply:
Huh, okay